src/frontend/app.ts

```diff
@@ -198,7 +198,6 @@ export default (octokitApp: App): Express => {
 	app.get(
 		"/github/setup",
 		csrfProtection,
-		oauth.checkGithubAuth,
```

Comment:
I wonder how did it work before? 

Am I correctly understand that we don't check this token anymore to support the case when we install the app from GitHub (without opening Jira) and get redirected here without GitHub Token?

Reply:
It worked before because it would go to `/github/setup`, see that it was missing the github token, go through the oauth flow to get it, redirect back to `/github/setup` then go through the process normally.  Nothing was broken per say, but we just don't need the github token at all for the github setup which is used after installing the app on the github side and needing to redirect back to jira.